src/snowflake/snowpark/udtf.py

```diff
@@ -908,41 +908,23 @@ def _do_register_udtf(
         _emit_ast: bool = True,
         **kwargs,
     ) -> UserDefinedTableFunction:
-
-        check_output_schema_type(output_schema)
-        check_imports_type(imports, "udtf-level")
-
-        # Capture original parameters.
-        ast, ast_id = None, None
-        if _emit_ast:
+        if "object_name" in kwargs:
```

Comment:
nit: maybe let's use one syntax here? (the is not None or in syntax?) No preference